src/static/js/modules/form-explainer.js

```diff
@@ -310,7 +306,11 @@ function filterExplainers ($currentTab, type) {
 function toggleScrollWatch() {
   $WINDOW.off('scroll.stickiness');
   if ($WINDOW.width() >= 600) {
-    $WINDOW.on('scroll.stickiness', debounce(updateStickiness, 20));
+    $WINDOW.on('scroll.stickiness', debounce(
+      function() {
+        var els =  formExplainer.getPageElements(formExplainer.currentPage);
```

Comment:
I cached this within the debounced function, but if it can be cached before scroll lmk and I will move it.